kafka-client/src/main/java/dev/responsive/kafka/api/async/internals/AsyncThreadPool.java

```diff
@@ -40,6 +41,7 @@ public class AsyncThreadPool {
   private final Logger log;
 
   private final String streamThreadName;
+  private final QueuedEvents queuedEvents;
```

Comment:
TODO: once we merge https://github.com/responsivedev/responsive-pub/pull/277 we need to pass this into the runnable so it can notify when a new event is dequeued